docs/scenarios_authoring/Enrichers.md

```diff
@@ -39,17 +39,34 @@ TTL (Time to Live) determines how long returned result is held in the cache of t
 
 DatabaseLookupEnricher is a specialized look up component; it returns all columns of a looked up record. In the example below the city table is looked up based on the id field. 
 
-![alt_text](img/databaseLookupEnricher.png "databaseLookupEnricher")
-
-
-
+![alt_text](img/databaseLookupEnricher.png "databaseLookup Enricher")
 
+  
 ## OpenAPI enricher
 
 [OpenAPI](https://swagger.io) is a specification for machine-readable interface files for describing, producing, consuming, and visualizing RESTful web services. Nussknacker can read definition of an OpenAPI interface and generate a component for the interaction with the given service.
 
 Once an OpenAPI component is configured in the Model it will become available in the Designer. Because Nussknacker can determine the definition of the service input parameters, the node configuration form will contain entry fields for these parameters. In the example below customer_id field is the input parameter to the openAPI service. 
 
-![alt_text](img/openApiEnricher.png "openApiEnricher")
+![alt_text](img/openApiEnricher.png "openAPI Enricher")
 
 Similarly, information about field names and types returned by the OpenAPI service are known to Designer; as the result Designer will hint them when openAPIResultVar variable is used in the SpEL expression. 
+     
+   
+## ML enricher
```

Comment:
Can you add `(Enterprise only)` suffix so it will be easy to recognize, that this part can't be find in normal quickstart tutorial etc. We are already doing similar things to distinguish between flink only components, see https://nussknacker.io/documentation/docs/scenarios_authoring/BasicNodes#unionmemo

Reply:
done